example/src/screens/Examples/AwareScrollViewStickyFooter/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useState } from 'react';
+import { LayoutChangeEvent, TextInput, View, Text, Button } from 'react-native';
+import { useResizeMode } from 'react-native-keyboard-controller';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { randomColor } from '../../../utils';
+
+import KeyboardAwareScrollView from './KeyboardAwareScrollView';
+import { StickyFooter } from './StickyFooter';
+import { styles } from './styles';
+
+export default function AwareScrollViewStickyFooter() {
+  useResizeMode();
+
+  const { bottom } = useSafeAreaInsets();
+
+  const [footerHeight, setFooterHeight] = useState(0);
+
+  const handleLayout = useCallback((evt: LayoutChangeEvent) => {
+    setFooterHeight(evt.nativeEvent.layout.height);
+  }, []);
+
+  return (
+    <View style={[styles.pageContainer, { paddingBottom: footerHeight }]}>
+      <KeyboardAwareScrollView
+        style={styles.container}
+        bottomOffset={footerHeight}
+        keyboardShouldPersistTaps="handled"
+      >
+        {new Array(10).fill(0).map((_, i) => (
+          <TextInput
+            key={i}
+            placeholder={`${i}`}
+            placeholderTextColor="black"
+            keyboardType={i % 2 === 0 ? 'numeric' : 'default'}
+            style={{
+              width: '100%',
+              height: 50,
+              backgroundColor: randomColor(),
+              marginTop: 50,
+            }}
+          />
+        ))}
+      </KeyboardAwareScrollView>
+      <StickyFooter
+        onLayout={handleLayout}
```

Comment:
Can you explain a little bit here? Why do we need to capture a layout of `StickyFooter` and then specify `paddingBottom` as `footerHeight`?

Can't we just have something like that:

```tsx
<View style={{flex: 1}}>
  <View style={{flexGrow: 1}}>
    <KeyboardAwareScrollView />
  </View>
  <StickyFooter />
</View>
```

The idea is that `StickyFooter` will take some height and a container for `ScrollView` will take a remaining space. Of course `StickyFooter` shouldn't have an `absolute` position, because otherwise it will not take its height into consideration.

Anyway if you have something against such approach and you see a downsides of it, please, share it here 😊 

What I currently don't like with such approach is that it may have a layout jump (especially on low end devices), because we re-calculate styles.